tests/sles4sap/saptune/mr_test.pm

```diff
@@ -18,11 +19,16 @@ use Utils::Systemd qw(systemctl);
 use strict;
 use warnings;
 use mr_test_lib qw(load_mr_tests);
+use publiccloud::ssh_interactive 'select_host_console';
 
 sub reboot_wait {
     my ($self) = @_;
 
-    $self->reboot;
+    # Do not reboot if testing is runing in public cloud instance,
```

Comment:
running

Reply:
More than they typo in `running`, I'd say one of the 2 instances of *ing needs to go. It could be either:
1. Do not reboot if testing in public cloud instance,
2. Do not reboot if test is running in public cloud instance.

Reply:
Done. Thank you all :)